pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -466,7 +466,7 @@ func (r *ReconcileLogStorage) Reconcile(request reconcile.Request) (reconcile.Re
 	if ls != nil {
 		hdler = utils.NewComponentHandler(log, r.client, r.scheme, ls)
 	} else {
-		hdler = utils.NewComponentHandler(log, r.client, r.scheme, installationCR)
+		hdler = utils.NewComponentHandler(log, r.client, r.scheme, managementClusterConnection)
```

Comment:
Are you getting input from Brian or Rene about using this for the owner ref?

Reply:
This seems like the right thing to do.